crates/workspace/src/dock.rs

```diff
@@ -165,6 +165,7 @@ pub struct Dock {
     position: DockPosition,
     panel_entries: Vec<PanelEntry>,
     is_open: bool,
+    use_full_width: bool,
```

Comment:
I don't think we want this to be configurable for each individual Dock. Rather, this seems like a variable that should be on the Workspace itself.